src/cloudkit.cs

```diff
@@ -489,6 +494,9 @@ interface CKErrorFields {
 
 	[iOS (8,0), Watch (3,0), TV (10,0), Mac (10,10, onlyOn64 : true)]
 	[BaseType (typeof (CKOperation))]
+	[Availability (Introduced = Platform.iOS_8_0 | Platform.Mac_10_10 | Platform.Watch_3_0 | Platform.TV_9_0, 
```

Comment:
The `Introduced` does not match the attributes above, just include `Deprecated`.
Also please re-write the message to follow our conventions.